### PR TITLE
DMC Scans: Fix selectors

### LINKED ISSUE #2472: DMC Scan always closed when view or download
### Source information

DMC Scan 1.4.9

### Source language

English

### Steps to reproduce

1. Open dmc scan, title "Jiang Du Chun", chapter 27
2. open via webview is fine
3. open chapter via app, show notification "closed"
4. download chapter, show notification "closed"

Help please

### Expected behavior

able to view and download chapter

### Actual behavior

Cannot open or download chapter

### Mihon/Tachiyomi version

0.16.5

### Android version

Android 12

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by sinkableShip**
Closes #2472

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt
Comment (AwkwardPeak7) [inline src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt:1]: add `1, 3` ratelimit to this source too, as it was requsested by the website owner

File: lib-multisrc/zeistmanga/build.gradle.kts [hunk 1]
```diff
@@ -2,4 +2,4 @@ plugins {
     id("lib-multisrc")
 }
 
-baseVersionCode = 9
+baseVersionCode = 10
```
Comment (vetleledaal) [inline lib-multisrc/zeistmanga/build.gradle.kts]: Since there's no real change in the base class, it should be fine to leave this as-is

Comment (AwkwardPeak7) [reply lib-multisrc/zeistmanga/build.gradle.kts:5]: agreed, no functionality changed in the base class so no need to bump it

**Comment by sinkableShip** [comment]
looks like some selector changed recently

File: src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt [hunk 2]
```diff
@@ -10,7 +10,7 @@ import org.jsoup.Jsoup
 
 class DMCScans : ZeistManga("DMC Scans", "https://didascans.blogspot.com", "en") {
     override val client = super.client.newBuilder()
-        .rateLimit(2)
+        .rateLimit(1, 3)
```
Comment (vetleledaal) [inline src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt]: Should add unit

File: src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt [hunk 3]
```diff
@@ -23,6 +23,12 @@ class DMCScans : ZeistManga("DMC Scans", "https://didascans.blogspot.com", "en")
 
     override val mangaDetailsSelectorGenres = "#labels > a[rel=tag]"
     override val mangaDetailsSelectorInfo = ".imptdt"
+    override val mangaDetailsSelectorDescription = "#desc"
+
+    // =========================== Chapter Feed =============================
+
+    override val chapterFeedRegex = """clwd.\.run\(["'](.*?)["']\)""".toRegex()
+    override val scriptSelector = ".epcheck > script"
```
Comment (vetleledaal) [inline src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt]: If selectors are constantly updated you need something more robust, like looking through every script tag and a more generic regex, or multiple regexes. Slightly different problem, but something semi-related can be seen [in this PR](https://github.com/keiyoushi/extensions-source/pull/2204).

**Review by vetleledaal — DISMISSED** [review]

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**